pkg/integrations/rawtemplate.go

```diff
@@ -135,11 +135,14 @@ func listComponents(pipelines map[pipeline.ID]PipelineConfig, listIDs func(Pipel
 // listReceivers lists the IDs of all the receivers found in pipelines
 func listReceivers(pipelines map[pipeline.ID]PipelineConfig) []component.ID {
 	return listComponents(pipelines, func(c PipelineConfig) []component.ID {
+		if c.Receiver == nil {
```

Comment:
This behavior doesn't seem tested. Should it be? (I know this was also in your other PR. Just thinking about it now).

Reply:
This is covered by the tests with pipelines without receivers. These tests panic without this check.